transformers/component_tests/cat/configured_transformer_test.go

```diff
@@ -136,22 +145,26 @@ var _ = Describe("Executing the transformer", func() {
 			var wardsResult test_helpers.MappingResWithAddress
 			err := db.Get(&wardsResult, `SELECT diff_id, header_id, address_id, usr AS key, wards.wards AS value FROM maker.wards`)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(wardsResult.AddressID).To(Equal(strconv.FormatInt(catAddressID, 10)))
+			Expect(wardsResult.AddressID).To(Equal(catAddressID))
 			test_helpers.AssertMapping(wardsResult.MappingRes, wardsDiff.ID, header.Id, strconv.FormatInt(userAddressID, 10), "1")
 		})
 	})
 
 	Describe("ilk", func() {
 		var (
-			ilk    string
-			ilkID  int64
-			ilkErr error
+			ilk                string
+			ilkID              int64
+			ilkErr             error
+			contractAddressID  int64
+			contractAddressErr error
```

Comment:
nbd but I feel like `ilkErr` and `contractAddressErr` could be declared within the `BeforeEach` so that they're not in scope for subsequent `It` statements - seems like we only care about them in the expectations on lines 165 + 167

Reply:
👍